DBZ-6618 Document usage of `${source.` prefix for table name formatting (#5631)
debezium/debezium: documentation/modules/ROOT/pages/connectors/jdbc.adoc

|[[jdbc-property-table-name-format]]<<jdbc-property-table-name-format, `+table.name.format+`>>
|`+${topic}+`
|Specifies a string that determines how the destination table name is formatted, based on the topic name of the event. +

> When the property is set to its default value, `+${topic}+`, when the connector reads an event from Kafka, it writes the event record to a destination table that is based on the name of the source topic . +

Reviewer: ```suggestion
When the property is set to its default value, `+${topic}+`, after the connector reads an event from Kafka, it writes the event record to a destination table with a name that matches the name of the source topic. +
```

 +
You can also use this property to dynamically influence the names of target tables, based on values in incoming events.
By applying this feature of the property, you can implement dynamic naming without having to write a custom single message transformation. +
 +

> To enable dynamic naming of output tables, set the value of this property to a pattern such as `+${source._field_}+`.

Reviewer: ```suggestion
To configure the property to dynamically generate the names of destination tables, set its value to a pattern such as `+${source._field_}+`.
```

> This type of pattern instructs the connector to extract values from the `source` block of the {prodname} change event for use in constructing the table name.

Reviewer: ```suggestion
When you specify this type of pattern, the connector extracts values from the `source` block of the {prodname} change event, and then uses those values to construct the table name.
```

For example, you might set the value of the property to the pattern `+${source.schema}_${source.table}+`.

> Based on this pattern, if the connector reads an event in which the source block contains the `schema` name value, `user`, and the `table` name value, `tab`, the connector writes the event record to a table with the name `user_tab`.

Reviewer: ```suggestion
Based on this pattern, if the connector reads an event in which the `schema` field in the source block contains the value, `user`, and the `table` field contains the value, `tab`, the connector writes the event record to a table with the name `user_tab`.
```

|[[jdbc-property-table-name-format]]<<jdbc-property-table-name-format, `+table.name.format+`>>
|`+${topic}+`

> |Specifies a string that determines how the destination table name is formatted, based on the topic name of the event.

Reviewer: ```suggestion
|Specifies a string that determines how the destination table name is formatted, based on the topic name of the event. +
```

The placeholder, `+${topic}+`, is replaced by the topic name. +
 +

> This configuration can also be specified using the `+${source._field_}+` pattern to use values from the Debezium change event's `source` information block for more dynamic formats.

Reviewer: ```suggestion
You can also use this property to dynamically influence the names of target tables, based on values in incoming events.
By applying this feature of the property, you can implement dynamic naming without having to write a custom single message transformation. +
 +
To enable dynamic naming of output tables, set the value of this property to a pattern such as `+${source._field_}+`.
This type of pattern instructs the connector to extract values from the `source` block of the {prodname} change event for use in constructing the table name.
```

Reviewer: I think that it makes sense to present the use case for enabling dynamic table naming via the property vs. using an SMT near the beginning of the description rather than mentioning it later. WDYT?

> For example, given a schema of `user` and the table name of `tab`, a value of `+${source.schema}_${source.table}+` would use a table name mapping of `user_tab`.

Reviewer: ```suggestion
For example, you might set the value of the property to the pattern `+${source.schema}_${source.table}+`.
Based on this pattern, if the connector reads an event in which the source block contains the `schema` name value, `user`, and the `table` name value, `tab`, the connector writes the event record to a table with the name `user_tab`.
```

> This allows each incoming event to dynamically influence the target table name without necessarily writing a custom single message transformation.

Reviewer: ```suggestion

```
Per my earlier comment, if we cover the idea in this sentence as in the earlier paragraph (new L829), we can drop it here.
